packages/polkadot/src/__snapshots__/assetHub.polkadot.test.ts.snap

```diff
@@ -0,0 +1,227 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`asset hub & polkadot > Teleport DOT from Asset Hub to Polkadot > balance on from chain 1`] = `
+{
+  "consumers": 0,
+  "data": {
+    "flags": "0x80000000000000000000000000000000",
+    "free": "(rounded 3000000000000)",
+    "frozen": 0,
+    "reserved": 0,
+  },
+  "nonce": 1,
+  "providers": 1,
+  "sufficients": 0,
+}
+`;
+
+exports[`asset hub & polkadot > Teleport DOT from Asset Hub to Polkadot > balance on to chain 1`] = `
+{
+  "consumers": 0,
+  "data": {
+    "flags": "0x80000000000000000000000000000000",
+    "free": 0,
```

Comment:
is it expected that the balance on to chain is zero?

Reply:
I was teleporting dots to Charlie, not Bob (which is the default). 
Yes, zero is expected for Charlie but now I changed it to Bob and the tests work properly. i.e. show 7*10^12 instead of 0.